.circleci/config.yml

```diff
@@ -440,9 +480,8 @@ workflows:
       - lint-workspace:
           requires:
             - install
-      - lint-styles:
-          requires:
-            - install
+      - bazel_stylelint:
+          <<: *filter_forks
```

Comment:
Why do we filter here for forked pull requests?

Reply:
good catch, I don't see a reason why this filter should be applied, will change

Reply:
just rebased proposed changes again